### PR TITLE
Issue 651

### LINKED ISSUE #651: Alternative typescript export workaround
I found an alternative workaround for the Typescript module exports workaround: 

```ts
import * as _module from './module';
import * as _module2 from './module2';

declare const exports: typeof _module & typeof _module2;

exports.foo
```

the advantage is that it is easy to define a whole module as part of the imported exports.

## Expected Behavior
n.a

## Actual Behavior
n.a

## Steps to Reproduce the Problem
n.a

1.
1.

## Specifications

- Node version (`node -v`):
- Version (`clasp -v`):
- OS (Mac/Linux/Windows):

### PR DESCRIPTION
**Pull request by PopGoesTheWza**
Fixes #651 

- [ ] `npm run test` succeeds.
- [ ] `npm run lint` succeeds.
- [ ] Appropriate changes to README are included in PR.

### CODE REVIEW

File: docs/typescript.md [hunk 1]
```diff
@@ -309,10 +318,10 @@ If you see outdated TypeScript types, you can help update them by contributing t
 ### How to Generate Types
 
 1. Fork [DefinitelyTyped/DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped)
-1. Run the types generator by following the instructions at [motemen/dts-google-apps-script](https://github.com/motemen/dts-google-apps-script)
-1. Copy the type files to your fork of `DefinitelyTyped/DefinitelyTyped`.
-1. View the diff (`git diff`), and make sure that the types look OK.
-1. Make a PR and ask [@grant](https://github.com/grant) for review.
+2. Run the types generator by following the instructions at [motemen/dts-google-apps-script](https://github.com/motemen/dts-google-apps-script)
```
Comment (grant) [inline docs/typescript.md]: Don't change this. Please revert.

Comment (PopGoesTheWza) [reply docs/typescript.md]: Not sure what is the cause of this change... grr

**Comment by PopGoesTheWza** [comment]
@grant is it OK now?

**Comment by grant** [comment]
Thanks!

**Pull request merged**